### PR TITLE
enh(release): Prepare Centreon Gorgone 20.10.4 RN

### PR DESCRIPTION
**Pull request by tuntoja**
## Description

Please include a short resume of the changes and what is the purpose of the PR.

Any relevant information should be added to help reviewers.

## Target serie

- [ ] 20.04.x
- [x] 20.10.x
- [ ] 21.04.x
- [ ] 21.10.x (master)

### CODE REVIEW

File: fr/releases/centreon-core.md [hunk 1]
```diff
@@ -647,6 +647,17 @@ with the –pool\_size X argument or -s X.
 
 ## Centreon Gorgone
 
+### 20.10.4
+
+### Bugfixes
+
+- [anomaly] Le host id était null dans les filtres de detection.
```
Comment (cg-tw) [inline fr/releases/centreon-core.md:654]: ID, détection

File: en/releases/centreon-core.md [hunk 2]
```diff
@@ -643,6 +643,17 @@ with the –pool\_size X argument or -s X.
 
 ## Centreon Gorgone
 
+### 20.10.4
+
+### Bugfixes
+
+- [anomaly] Host id was null in detection filters
```
Comment (cg-tw) [inline en/releases/centreon-core.md:650]: ID

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: en/releases/centreon-core.md (hunk 2 above)
Comment (tuntoja) [inline en/releases/centreon-core.md:650]:
```suggestion
- [anomaly] Host ID was null in detection filters
```

File: fr/releases/centreon-core.md [hunk 3]
```diff
@@ -647,6 +647,17 @@ with the –pool\_size X argument or -s X.
 
 ## Centreon Gorgone
 
+### 20.10.4
+
+### Bugfixes
```
Comment (tuntoja) [inline fr/releases/centreon-core.md:652]:
```suggestion
#### Correctifs
```

File: fr/releases/centreon-core.md (hunk 1 above)
Comment (tuntoja) [inline fr/releases/centreon-core.md:654]:
```suggestion
- [anomaly] Le host ID était null dans les filtres de détection.
```

File: fr/releases/centreon-core.md [hunk 4]
```diff
@@ -647,6 +647,17 @@ with the –pool\_size X argument or -s X.
...
+
+- [anomaly] Le host id était null dans les filtres de detection.
+
+### Enhancements
```
Comment (tuntoja) [inline fr/releases/centreon-core.md:656]:
```suggestion
#### Améliorations
```

File: en/releases/centreon-core.md [hunk 5]
```diff
@@ -643,6 +643,17 @@ with the –pool\_size X argument or -s X.
...
+
+### Enhancements
```
Comment (tuntoja) [inline en/releases/centreon-core.md:652]:
```suggestion
#### Enhancements
```

File: en/releases/centreon-core.md [hunk 6]
```diff
@@ -643,6 +643,17 @@ with the –pool\_size X argument or -s X.
 
 ## Centreon Gorgone
 
+### 20.10.4
+
+### Bugfixes
```
Comment (tuntoja) [inline en/releases/centreon-core.md:648]:
```suggestion
#### Bugfixes
```

**Review by cg-tw — APPROVED** [review]

File: en/releases/centreon-core.md [hunk 7]
```diff
@@ -643,6 +643,17 @@ with the –pool\_size X argument or -s X.
 
 ## Centreon Gorgone
 
+### 20.10.4
+
+#### Bugfixes
+
+- [anomaly] Host ID was null in detection filters
```
Comment (omercier) [inline en/releases/centreon-core.md:650]:
```suggestion
- [Anomaly] Host ID was null in detection filters
```

File: en/releases/centreon-core.md [hunk 8]
```diff
@@ -643,6 +643,17 @@ with the –pool\_size X argument or -s X.
...
+
+#### Enhancements
+
+- add logging option to better handle logging that could cause SQLite database to grow too large
```
Comment (omercier) [inline en/releases/centreon-core.md:654]:
```suggestion
- Add logging option to better handle logs that could cause SQLite database to grow too large
```

File: en/releases/centreon-core.md [hunk 9]
```diff
@@ -643,6 +643,17 @@ with the –pool\_size X argument or -s X.
...
+- added dependency for perl-Libssh-Session-0.8
```
Comment (omercier) [inline en/releases/centreon-core.md:655]:
```suggestion
- Added dependency for perl-Libssh-Session-0.8
```

File: fr/releases/centreon-core.md [hunk 10]
```diff
@@ -647,6 +647,17 @@ with the –pool\_size X argument or -s X.
 
 ## Centreon Gorgone
 
+### 20.10.4
+
+#### Correctifs
+
+- [anomaly] Le host ID était null dans les filtres de détection.
```
Comment (omercier) [inline fr/releases/centreon-core.md:654]:
```suggestion
- [Anomaly] Le host ID était null dans les filtres de détection.
```

File: fr/releases/centreon-core.md [hunk 11]
```diff
@@ -647,6 +647,17 @@ with the –pool\_size X argument or -s X.
...
+
+#### Améliorations
+
+- ajout d'une option de journalisation pour éviter une taille trop grande de la base SQLite
```
Comment (omercier) [inline fr/releases/centreon-core.md:658]:
```suggestion
- Ajout d'une option de journalisation pour éviter une taille trop grande de la base SQLite
```

File: fr/releases/centreon-core.md [hunk 12]
```diff
@@ -647,6 +647,17 @@ with the –pool\_size X argument or -s X.
...
+- ajout de la dépendance pour perl-Libssh-Session-0.8
```
Comment (omercier) [inline fr/releases/centreon-core.md:659]:
```suggestion
- Ajout de la dépendance pour perl-Libssh-Session-0.8
```

**Review by omercier — CHANGES_REQUESTED** [review]

**Review by mdebauchez — APPROVED** [review]

**Review by thiuyendang — APPROVED** [review]

File: en/releases/centreon-core.md [hunk 13]
```diff
@@ -643,6 +643,17 @@ with the –pool\_size X argument or -s X.
 
 ## Centreon Gorgone
 
+### 20.10.4
+
```
Comment (tuntoja) [inline en/releases/centreon-core.md:647]:
```suggestion

`June 10, 2021`

```

File: fr/releases/centreon-core.md [hunk 14]
```diff
@@ -647,6 +647,17 @@ with the –pool\_size X argument or -s X.
 
 ## Centreon Gorgone
 
+### 20.10.4
+
```
Comment (tuntoja) [inline fr/releases/centreon-core.md:651]:
```suggestion

`10 juin 2021`

```

**Pull request merged**